src/App.js

```diff
@@ -49,12 +49,12 @@ const URLS = {
 }
 
 const HEADER_URL_DEFAULTS = {
-  "acctrequest": "",
-  "acctupdate": "",
-  "passreset": "",
-  "passchg": "",
-  "gmui": "",
-  "search": "",
+  "ivo://cadc.nrc.ca/acctrequest": "",
```

Comment:
There are constants defined for some of these values in `CanfarNavbar.js`. Should `App.js` use those constants?

Reply:
The `App.js` file uses the `headerURLs` variable to distribute to other React components, so no, it's just storing them to use in building other things.